contracts/test/performanceFee/FlywheelPerformanceFee.t.sol

```diff
@@ -176,4 +179,43 @@ contract FlywheelPerformanceFeeTest is BaseTest {
     );
     assertEq(dddFlywheel.rewardsAccrued(feeRecipient), 0, "feeRecipient rewardsAccrued should be 0");
   }
+
+  function testMoonbeamAllFlywheelsFeeRecipient() public debuggingOnly fork(MOONBEAM_MAINNET) {
+    _testAllFlywheelsFeeRecipient();
+  }
+
+  function testPolygonAllFlywheelsFeeRecipient() public debuggingOnly fork(POLYGON_MAINNET) {
+    _testAllFlywheelsFeeRecipient();
+  }
+
+  function testBscAllFlywheelsFeeRecipient() public debuggingOnly fork(BSC_MAINNET) {
+    _testAllFlywheelsFeeRecipient();
+  }
+
+  function testFantomAllFlywheelsFeeRecipient() public debuggingOnly fork(FANTOM_OPERA) {
+    _testAllFlywheelsFeeRecipient();
+  }
+
+  function testEvmosAllFlywheelsFeeRecipient() public debuggingOnly fork(EVMOS_MAINNET) {
+    _testAllFlywheelsFeeRecipient();
+  }
+
+  function testArbitrumAllFlywheelsFeeRecipient() public debuggingOnly fork(ARBITRUM_ONE) {
+    _testAllFlywheelsFeeRecipient();
+  }
+
+  function _testAllFlywheelsFeeRecipient() internal {
+    FusePoolDirectory fpd = FusePoolDirectory(ap.getAddress("FusePoolDirectory"));
+    (, FusePoolDirectory.FusePool[] memory pools) = fpd.getActivePools();
+
+    for (uint8 i = 0; i < pools.length; i++) {
+      ComptrollerFirstExtension poolExt = ComptrollerFirstExtension(pools[i].comptroller);
+      address[] memory fws = poolExt.getRewardsDistributors();
+      for (uint256 j = 0; j < fws.length; j++) {
+        address fr = MidasFlywheelCore(fws[j]).feeRecipient();
+        if (fr != 0x27521eae4eE4153214CaDc3eCD703b9B0326C908) emit log_named_address("flywheel fr", fws[j]);
+        assertEq(fr, 0x27521eae4eE4153214CaDc3eCD703b9B0326C908, "fee recipient not correct");
```

Comment:
Should we maybe add the deployer's address to the AP?

Reply:
sounds good, let's add it... we should also add the other roles when we are ready to deploy the admin roles separation